.azure-pipelines/all.yml

```diff
@@ -135,10 +138,10 @@ jobs:
       displayName: Puma
       os: linux
     - checkName: purefa
-      displayName: PureFA
+      displayName: pureFA
       os: linux
     - checkName: purefb
-      displayName: PureFB
+      displayName: pureFB
```

Comment:
Run `ddev -x validate ci --fix` to fix this automatically.

Reply:
@steveny91, @fanny-jiang  - This .azure-pipelines/all.yml has been a big problem from the beginning. This purefa and purefb has nothing to do with my code and I dont know how it got changed. And I believe I had changed it based on a comment earlier. Now its come back again. This is what my all.yml file looks like for these lines

    - checkName: purefa
      displayName: purefa
      os: linux
    - checkName: purefb
      displayName: purefb

Do you want me to change this to?

    - checkName: purefa
      displayName: pureFA
      os: linux
    - checkName: purefb
      displayName: pureFB

If not, can you type in what this needs to be changed to so that I can get over this problem once and for all?